contracts/BaseRelayRecipient.sol

```diff
@@ -47,19 +47,8 @@ abstract contract BaseRelayRecipient is IRelayRecipient {
      * signing or hashing the
      */
     function _msgData() internal override virtual view returns (bytes memory ret) {
-        if (msg.data.length >= 24 && isTrustedForwarder(msg.sender)) {
-            // At this point we know that the sender is a trusted forwarder,
-            // we copy the msg.data , except the last 20 bytes (and update the total length)
-            assembly {
-                let ptr := mload(0x40)
-                // copy only size-20 bytes
-                let size := sub(calldatasize(),20)
-                // structure RLP data as <offset> <length> <bytes>
-                mstore(ptr, 0x20)
-                mstore(add(ptr,32), size)
-                calldatacopy(add(ptr,64), 0, size)
-                return(ptr, add(size,64))
-            }
+        if (msg.data.length >= 20 && isTrustedForwarder(msg.sender)) {
```

Comment:
Why did you change 24 to 20?

Do we support calling fallback functions?

Reply:
GSN doesn't support callback functions, but there is no reason for the forwarder not to.

Reply:
Ok. Why this contract is not in a Forwarder folder then? Just asking, if you agree it is part of Forwarder project will add a jira task to put it there.

Reply:
Right. It should be moved